src/components/NavBar/index.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import { Layout, Input, Space, Dropdown, Menu, Avatar, Button, Form } from 'antd';
+
+import { UserOutlined, SearchOutlined } from '@ant-design/icons';
+
+import { useAuth } from 'hooks/Hooks';
+
+import routes from 'constants/routes';
+
+import { useHistory } from 'react-router-dom';
+
+import './styles.scss';
+
+import snippetslogo from 'assets/img/logoblack.svg';
+
+const { Header } = Layout;
+
+function NavBar() {
+  const { logout } = useAuth();
+  const history = useHistory();
+
+  async function handleLogout() {
+    try {
+      await logout();
+      history.push(routes.LOGIN);
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.log(error.message);
+    }
+  }
+
+  const menu = (
+    <Menu onClick={handleLogout}>
+      <Menu.Item key="1">Profile</Menu.Item>
+      <Menu.Item key="2">Logout</Menu.Item>
+    </Menu>
+  );
+
+  // eslint-disable-next-line no-console
+  const onSearch = (value) => console.log(value);

```

Comment:
lets do functions instead of arrow functions for methods inside the functional component